Shapes/shapes/v2/graph_map.ttl

```diff
@@ -0,0 +1,138 @@
+###############################################################################
+# RML Graph Map and graph shortcut shape                                      #
+# Copyright Dylan Van Assche, IDLab - UGent - imec (2020 - 2023)              #
+###############################################################################
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix schema: <http://schema.org/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rml: <http://w3id.org/rml/core/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> . 
+
+schema:RMLGraphMapShape
+    a sh:NodeShape ;
+    sh:ignoredProperties (rdf:type) ;
+    sh:name "rml:graphMap" ;
+    sh:description """
+    Represents a graph map.
+    """ ;
+    sh:message """
+    rml:GraphMap must specify an rml:template, rml:reference or rml:constant 
+    with the IRI of the Named Graph.
+    """ ;
+
+    # Exactly one rml:template or one rml:constant is required
+    sh:property [
+        sh:path [sh:alternativePath (rml:template rml:constant)] ;
+        sh:name "rml:template/rml:reference/rml:constant" ;
+        sh:description """
+        Exactly one rml:template or one rml:reference or one rml:constant is
+        required.
+        """ ;
+        sh:message """
+        Exactly one rml:template or one rml:reference or one rml:constant is
+        required.
+        """ ;
+        sh:minCount 1 ;
+        sh:maxCount 1 ;
+    ] ;
+
+    # rml:template
+    sh:property [
+        sh:path rml:template ;
+        sh:name "rml:template" ;
+        sh:description """
+        A template (format string) to specify how to generate a value for a 
+        subject, predicate, or object, using one or more columns from a logical
+        table row or iterator.
+        """ ;
+        sh:message """
+        rml:template must be a string.
+        """ ;
+        sh:nodeKind sh:Literal ;
+        sh:datatype xsd:string ;
+    ] ;
+
+    # rml:reference
+    sh:property [
```

Comment:
that's the `ReferencePropertyShape` mentioned in my other comment right?

```schema:RMLGraphMapShape sh:property sh:and ( :ReferencePropertyShape :LiteralPropertyShape).

:ReferencePropertyShape sh:path rml:reference ; sh:node ReferenceNodeShape.

:LiteralPropertyShape sh:path rml:termType; sh:in (rml:IRI rml:BlankNode)`

(I hope it doesn't contain mistakes but it might need some refinement)